.github/workflows/create-release.yml

```diff
@@ -1,27 +1,61 @@
-name: Create release assets
+name: Create Atlantis release
 
-on:
-  push:
-    branches: [ master ]
+on: workflow_dispatch
```

Comment:
1. Trigger on new CLI release via GH release workflow.
2. Trigger manually (automate later) when new Atlantis release is available.